pyrit/prompt_converter/leetspeak_converter.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pyrit.models import PromptDataType
+from pyrit.prompt_converter import PromptConverter, ConverterResult
+import random
+
+
+class LeetspeakConverter(PromptConverter):
+    """Converts a string to a leetspeak version"""
+
+    def __init__(self) -> None:
+        self.leet_substitutions = {
+            "a": ["4", "@", "/\\", "@", "^", "/-\\"],
+            "b": ["8", "6", "13", "|3", "/3", "!3"],
+            "c": ["(", "[", "<", "{"],
+            "e": ["3"],
+            "g": ["9"],
+            "i": ["1", "!"],
+            "l": ["1", "|"],
+            "o": ["0"],
+            "s": ["5", "$"],
+            "t": ["7"],
+            "z": ["2"],
+        }
+
+    def convert(self, *, prompt: str, input_type: PromptDataType = "text") -> ConverterResult:
+        """
+        Simple converter to generate leatspeak version of a prompt.
+        Since there are multiple character variations, this is non-deterministic
```

Comment:
```suggestion
        Since there are multiple character variations, this is non-deterministic.
```
I know, very nitpicky 🙂